### ISSUE TITLE
Allow the ability to duplicate entries for documents or bulk upload

### ISSUE THREAD

**Issue by lyndoer** [massgov/feature-requests#238]
**Is your feature request related to a problem? Please describe.**
No, the issue is not related to a problem.

**Describe the solution you'd like**
Users should be able to upload files in bulk or, if that isn't possible, we should be able to duplicate document entries.  I have found myself in several situations where I have had to upload several documents that were very similar (e.g. same author, tags, labels, etc.) but would require different uploads.  It takes FOREVER to do and would be much easier if I could bulk upload and edit the document details in table format (e.g. if I needed to upload minutes from a particular committee for a year, I could upload them all, reuse most of the details and change the month and year in the title).

**Additional context**
This is a significant time sink so a feature upgrade would save so much!

**Comment by karmenchong** [comment]
I agree with this! I upload ABCC decisions, which can be as many as 30+ PDFs at a time, and I have to use 3 labels per PDF. This is a very manual and time consuming task that could be streamlined.

**Comment by joeg8612** [comment]
Good idea. The current code doesn't support clone of documents but it is something we could look into in future.

**Comment by joeg8612** [comment]
We now support cloning of documents. We currently don't have bulk upload as metadata is required for each document.
